castle/src/main/java/io/castle/android/Castle.java

```diff
@@ -77,13 +90,13 @@ private void registerLifeCycleCallbacks(Application application) {
 
         // Check and track Application Installed or Application Updated.
         if (previousBuild == -1) {
-            track("Application Installed");
+            custom("Application Installed");
```

Comment:
Didn't we want to call this `customWithName` in Obj-C and `custom(name:properties:)` in Swift?

Reply:
This is the equivalent of that, since we don't have named parameters and the `withName` is a objective-c convention